Makefile

```diff
@@ -0,0 +1,353 @@
+# Determine this makefile's path.
+# Be sure to place this BEFORE `include` directives, if any.
+THIS_FILE := $(lastword $(MAKEFILE_LIST))
+
+include vars.mk
+
+#################################
+# Documentation targets
+#################################
+
+.PHONY: publish-docs
+docs-publish: ## build and publish docs
+	docker run --rm -it -v $(PWD):/knora -v $(HOME)/.ivy2:/root/.ivy2 -v $HOME/.ssh:/root/.ssh sbt-paradox /bin/sh -c "cd /knora && git config --global user.email $(GIT_EMAIL) && sbt docs/ghpagesPushSite"
```

Comment:
You have `$(HOME)` and `$HOME`, is that correct?

Reply:
I don't think that it matters, but I've added parentheses. Now both should be resolved by `make`.